README.md

```diff
@@ -1,7 +1,7 @@
 # rake js:routes
 
 [![Gem](https://img.shields.io/gem/v/js_rails_routes.svg?maxAge=2592000)](https://rubygems.org/gems/js_rails_routes)
-[![Build Status](https://travis-ci.org/increments/js_rails_routes.svg?branch=master)](https://travis-ci.org/increments/js_rails_routes)
+![Build Status](https://github.com/increments/js_rails_routes/actions/workflows/test.yml/badge.svg?branch=master)
```

Comment:
Now, master branch does not have build result.
For check result, see result of this branch.
https://github.com/increments/js_rails_routes/actions/workflows/test.yml/badge.svg?branch=move-github-actions